
Header header

uint32 sender_id 
Vertex sender_loc
Vertex sender_goal
